### ISSUE TITLE
[bitnami/memcached] Add Memecache TLS support

### ISSUE THREAD

**Issue by ntimo** [bitnami/containers#47418]
### Name and Version

bitnami/memcache:1.6.21

### What is the problem this feature will solve?

Securing memcache with tls.

### What is the feature you are proposing to solve the problem?

Memcache should be build with TLS support.
Docs: https://github.com/memcached/memcached/wiki/TLS

**Comment by juan131** [comment]
Thanks so much @ntimo !! I just created an internal task to adapt our compilation recipe so the memcached binary is compiled with TLS support. I'll let you know as soon as there's a container release with a binary including these changes.

**Comment by juan131** [comment]
Hi @ntimo 

We just released a new image revision (`1.6.21-debian-11-r84`) with TLS support, see:

```console
$ docker run --rm bitnami/memcached:1.6.21-debian-11-r84 memcached --help
(...)
   - ssl_chain_cert:      certificate chain file in PEM format
   - ssl_key:             private key, if not part of the -ssl_chain_cert
   - ssl_keyformat:       private key format (PEM, DER or ENGINE) (default: PEM)
   - ssl_verify_mode:     peer certificate verification mode, default is 0(None).
                          valid values are 0(None), 1(Request), 2(Require)
                          or 3(Once)
   - ssl_ciphers:         specify cipher list to be used
   - ssl_ca_cert:         PEM format file of acceptable client CA's
   - ssl_wbuf_size:       size in kilobytes of per-connection SSL output buffer
                          (default: 16)
   - ssl_session_cache:   enable server-side SSL session cache, to support session
                          resumption
   - ssl_kernel_tls:      enable kernel TLS offload
   - ssl_min_version:     minimum protocol version to accept (default: tlsv1.2)
                          valid values are 0(tlsv1.0), 1(tlsv1.1), 2(tlsv1.2), or 3(tlsv1.3).
```
